dolly-backend-app/src/main/java/no/nav/dolly/bestilling/arenaforvalter/ArenaForvalterClient.java

```diff
@@ -202,9 +208,29 @@ private void sendArenadata(ArenaNyeBrukere arenaNyeBrukere, StringBuilder status
         }
     }
 
+    private ArenaNyeBrukere filtrerEksisterendeBrukere(ArenaNyeBrukere arenaNyeBrukere) {
+        List<String> eksisterendeBrukere = new ArrayList<>();
+
+        ResponseEntity<ArenaArbeidssokerBruker> ident = arenaForvalterConsumer.getIdent(arenaNyeBrukere.getNyeBrukere().get(0).getPersonident());
+        if (nonNull(ident)) {
+            log.info("Liste over eksisterende brukere: {}", Json.pretty(ident.getBody()));
+        }
+
+        arenaNyeBrukere.getNyeBrukere().forEach(arenaNyBruker -> {
+            if (isNull(arenaNyBruker.getKvalifiseringsgruppe()) && isNull(arenaNyBruker.getUtenServicebehov())) {
+                eksisterendeBrukere.add(arenaNyBruker.getPersonident());
+            }
+        });
+
+        return new ArenaNyeBrukere(arenaNyeBrukere.getNyeBrukere().stream()
+                .filter(arenaNyBruker -> eksisterendeBrukere.stream()
+                        .noneMatch(eksisterende -> arenaNyBruker.getPersonident().equals(eksisterende)))
+                .collect(Collectors.toList()));
```

Comment:
Tror ikke det er nødvendig med en egen variabel for `eksisterende`. Kan nok bruke filter direkte. Noe slikt;
```
        return new ArenaNyeBrukere(arenaNyeBrukere.getNyeBrukere().stream()
                .filter(arenaNyBruker -> (!isNull(arenaNyBruker.getKvalifiseringsgruppe()) ||  !isNull(arenaNyBruker.getUtenServicebehov())) ))
                .collect(Collectors.toList()));
```

Reply:
Godt poeng, skal forbedre 😄